package.json

```diff
@@ -2,11 +2,11 @@
 	"about": {
 		"name": "Titanium Command-Line Interface",
 		"author": "Appcelerator",
-		"copyright": "Copyright (c) 2012-2020, Appcelerator, Inc.  All Rights Reserved.",
+		"copyright": "Copyright (c) 2012-2022, TiDev, Inc. All Rights Reserved.",
```

Comment:
I have been waiting to discuss the copyright before changing it. I'm not a lawyer, so I don't know if we can change it, or if we can, maybe we just reset the year to 2022?